### ISSUE TITLE
Compile sass/scss to css failed with weppy-assets.

### ISSUE THREAD

**Issue by laozhu** [gi0baro/weppy-assets#1]
I got vendor.css successful, theme.css failed.
## vendor.css

``` python
vendor = app.ext.Assets.css(
    "vendor/bootstrap/dist/css/bootstrap.min.css",
    "vendor/bootstrap/dist/css/bootstrap-theme.min.css",
    "vendor/bootstrap-datepicker/dist/css/bootstrap-datepicker.min.css",
    "vendor/font-awesome/css/font-awesome.min.css",
    "vendor/prism/themes/prism-tomorrow.css",
    output="vendor.css"
).minify()

app.ext.Assets.register("base_css", vendor)
```

**OUTPUT**

``` html
 <link rel="stylesheet" href="/static/dist/vendor.css?1416e949" type="text/css">
```
## theme.css

``` python
theme = app.ext.Assets.css(
    "styles/base.scss",
    output="theme.css"
).minify()

app.ext.Assets.register("user_css", theme)
```

**OUTPUT**

![image](https://cloud.githubusercontent.com/assets/1212457/14233656/1b9171cc-fa01-11e5-96f8-da527d7aaf65.png)

I didn't get what's wrong with my code, beg for some help here.

**Comment by gi0baro** [comment]
@laozhu this seems to be a problem with the python sass library.
I can check tonight if the filters need an update and if so, release an update. Then if the problem persist, we can only ask to the developers of the library (https://github.com/dahlia/libsass-python).

Will keep you updated.

**Comment by laozhu** [comment]
More details about my environment.
I worked with virtualenvwrapper, this is my requirements for project.

``` shell
[publog]publog » pip freeze
click==6.4
CoffeeScript==1.1.2
cssmin==0.2.0
jsmin==2.2.1
libsass==0.11.0
pyaes==1.3.0
pyDAL==16.3
PyExecJS==1.3.1
PyYAML==3.11
six==1.10.0
weppy==0.6.3
weppy-Assets==0.3.1
weppy-Haml==0.3
```

I played libsass-python in ipython with above virtualenv, It's all right.

![image](https://cloud.githubusercontent.com/assets/1212457/14233737/abe0253c-fa03-11e5-9208-ebedbba1b523.png)

Thanks for your quick reply :+1:

**Comment by gi0baro** [comment]
@laozhu I've just updated the webassets library with 9b1c8b1 on master.
Can you try installing the latest mater and check if the problem persists?

**Comment by laozhu** [comment]
@gi0baro I install master version and compiled mixed sass and css, it works all right now :+1: 

``` shell
[pubean]pubean » pip freeze
click==6.4
CoffeeScript==1.1.2
cssmin==0.2.0
jsmin==2.2.1
libsass==0.11.0
pyaes==1.3.0
pyDAL==15.12
PyExecJS==1.3.1
PyYAML==3.11
six==1.10.0
weppy==0.6.3
weppy-Assets==0.4.dev0
weppy-Haml==0.3
```

**Comment by gi0baro** [comment]
Ok then, will release 0.4 later today.

**Comment by gi0baro** [comment]
Published 0.4, I'm closing this.
